File: backend/models/member.py
from ..extensions import db

class Member(db.Model):
    __tablename__ = 'members'
    id = db.Column(db.Integer, primary_key=True)
    full_name = db.Column(db.String(150), nullable=False)
    email = db.Column(db.String(120), unique=True, nullable=False)
    joined_date = db.Column(db.Date, nullable=False)

File: backend/src/models/member.js
const { DataTypes } = require('sequelize');
module.exports = (sequelize) => {
  return sequelize.define('Member', {
    id: { type: DataTypes.UUID, primaryKey: true, defaultValue: DataTypes.UUIDV4 },
    firstName: { type: DataTypes.STRING, allowNull: false },
    lastName: { type: DataTypes.STRING, allowNull: false },
    email: { type: DataTypes.STRING, allowNull: false, unique: true }
  }, { tableName: 'Members', timestamps: true });
};
